sdk/servicebus/azure-messaging-servicebus/migration-guide.md

```diff
@@ -395,6 +395,39 @@ ServiceBusSessionReceiverClient sessionClient = new ServiceBusClientBuilder()
 
 ServiceBusReceiverClient receiverClient = sessionClient.acceptSession("my-session-id");
 ```
+
+## Working with Service Access Signature Token
+
+Previously, in `azure-servicebus`, you could create a new instance from the given connection string and entity path as
+shown below. 
+
+```java
+String SHARED_ACCESS_SIGNATURE_TOKEN = "<Shared Access Signatures for specific Resource>";
+String connectionString = "Endpoint=sb://namespace_DNS_Name;SharedAccessSignatureToken=" + SHARED_ACCESS_SIGNATURE_TOKEN;
```

Comment:
`namespace_DNS_Name` looks a new term to users. I suggest we use the same term as in other docs.